[WIP] Add future issue templates for github (#9438)
qgis/QGIS: .github/bug_report.md

---
name: Bug report
about: Create a bug report to help us improve QGIS.
title: ''
labels: ''
assignees: ''

> ---

Reviewer: What is this first section above? Do we need to fill manually labels and assignees?
Do we need to keep `''`?
Sorry if I missed something

Author: Hi, the first section is the metadata section for github's template where you can put defaults title, labels and assignments. I'all add the default label 'bug report' at least.

**Checklist before submitting**

> - [ ] Search in the issues, stackexchange, forums before if the issue already exists

Reviewer 2: ```suggestion
- [ ] Search through existing issue reports and gis.stackexchange.com to check whether the issue already exists
```

What forums? I'd drop that part.

Author: fixed

- [ ] Test with a [clean new user profile](https://docs.qgis.org/testing/en/docs/user_manual/introduction/qgis_configuration.html?highlight=profile#working-with-user-profiles). Do not add third party plugins at that step

> - [ ] Create light and portable sample dataset and project file that reproduce the issue

Reviewer 2: ```suggestion
- [ ] Create a light and self-contained sample dataset and project file which demonstrates the issue
```

Author: fixed, sorry for the poor english

> If the issue concerns a **third party plugin**, please raise the issue in the dedicated bug tracker mentionned in plugin description.

Reviewer 3: s/mentionned/mentioned

Author: fixed

> If the issue concerns a **third party plugin**, please raise the issue in the dedicated bug tracker mentionned in plugin description.

Reviewer 2: ```suggestion
If the issue concerns a **third party plugin** then it **cannot** be fixed by the QGIS team. Please raise your issue in the dedicated bug tracker for that specific plugin (as listed in the plugin's description).
```

Author: done

**Describe the bug**

> A clear and concise description of what the bug is. Tip

Reviewer: IMHO, there is a lot of strings to remove when we fill a bug. I guess we prefer not having these strings in the final report. Can we put this help as comment? But comments are also harder for beginners.
Check the result and the raw file of https://github.com/inasafe/inasafe/blob/develop/.github/PULL_REQUEST_TEMPLATE.md

Author: Good point, I used the default github template to start. I'll try to simplify all this

Reviewer: So it seems you are not going to use comments 😉

Author: > So it seems you are not going to use comments wink

In fact I didn't get what you mean Etienne, can you be more explicit?

Reviewer: You can put strings as markdowns comments. They are not displayed by default. Only when editing the markdown. Check my link how it's displayed and check the raw file. You will see some markdowns comments only when you click on the raw file. It's much much better IMHO because these strings are not displayed when we read the issue, only when we edit it. So it's less "noise" for readers.

Author: @Gustry understood now :) ID didn't see it directly in your link as I didn't switch to edit mode . 
Let's do that !

Author: Done

**To Reproduce**
Steps, sample datasets and qgis project file to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**

> If applicable, add screenshots to help explain your problem.

Reviewer 2: ```suggestion
If applicable, add a screencast or screenshots to help explain your problem.
```

(I find screencasts are much easier to reproduce!)
